src/mono/sample/wasm/browser-advanced/Wasm.Advanced.Sample.csproj

```diff
@@ -16,8 +16,6 @@
     <!-- for callspec see diagnostics-tracing.md -->
     <WasmProfilers>browser:callspec=N:Sample</WasmProfilers>
 
-    <!-- Put "framework" (dotnet.js, dlls, etc) files directly into the AppBundle -->
-    <WasmRuntimeAssetsLocation>./</WasmRuntimeAssetsLocation>
```

Comment:
Does `WasmRuntimeAssetsLocation` work for WASM SDK ?

Reply:
No https://github.com/dotnet/runtime/issues/90979